stopify-continuations/src/runtime/eagerRuntime.ts

```diff
@@ -17,7 +17,8 @@ export class EagerRuntime extends common.Runtime {
     throw new common.Capture(f, [...this.eagerStack]);
   }
 
-  makeCont(stack: common.Stack) {
+  // TODO(rachit): savedStack should be used here.
+  makeCont(stack: common.Stack, savedStack: common.Stack) {
```

Comment:
So, why is it okay to ignore `savedStack`?

Reply:
Because eager doesn’t support deep stacks.